components/html-block/html-block.js

```diff
@@ -9,7 +9,10 @@ export const htmlBlockContentStyles = css`
 	.d2l-html-block-compact {
 		font-size: 0.8rem;
 		font-weight: 400;
-		line-height: 1.2rem;
+		line-height: 1.5;
+	}
+	.d2l-html-block-rendered {
+		line-height: 1.47;
```

Comment:
1.47 ~= 1.4rem / 0.95rem

Reply:
Might be worth putting these calculations in a comment so we can remember how they're derived and in case we ever need to re-calculate them based on changes to our root sizes.